Makefile

```diff
@@ -93,3 +93,7 @@ release-manager-snapshot:
 .PHONY: release-manager-release
 release-manager-release:
 	./_beats/dev-tools/run_with_go_ver $(MAKE) release
+
+.PHONY: bench
+bench:
+	@go test -benchmem -run=XXX -benchtime=100ms -bench='.*Processor$$' ./processor/...
```

Comment:
I wouldn't have a `Processor` regex because if we ever create a benchmark somewhere else (or just rename those tests) then `make bench` would silently skip them.